packages/unit-testing/stubs/workbook/Expression.js

```diff
@@ -0,0 +1,48 @@
+define([], function () {
+	/**
+	 * @class Expresion
+	 * @classDescription An expression. An expression can be used when you create a pivot definition, a data dimension item, a measure, a conditional filter, or a constant.
+	 * @constructor
+	 * @protected
+	 *
+	 * @since 2021.2
+	 */
+	function Expresion() {
```

Comment:
Misspelling... should be **Expression**

Reply:
Corrected. Thank you Darren! @Extend-Apps